src/components/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+interface State {
+  today: Date,
+}
+
+interface Props {
+  clockName: string,
+}
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      window.console.info(this.state.today.toUTCString().slice(-12, -4));
```

Comment:
will be great to create separate function for refactoring date in needed format, as you used the same functionality twice in your code